tests/test.nf.test.snap

```diff
@@ -127,9 +127,9 @@
                 "tables/annotation/Mus_musculus.anno.tsv",
                 "tables/differential",
                 "tables/differential/treatment_mCherry_hND6_.deseq2.results.tsv",
-                "tables/differential/treatment_mCherry_hND6_.deseq2.results_filtered.tsv",
+                "tables/differential/treatment_mCherry_hND6__filtered.tsv",
                 "tables/differential/treatment_mCherry_hND6_sample_number.deseq2.results.tsv",
-                "tables/differential/treatment_mCherry_hND6_sample_number.deseq2.results_filtered.tsv",
+                "tables/differential/treatment_mCherry_hND6_sample_number_filtered.tsv",
```

Comment:
Why has the method been removed from the outputs?

Reply:
I replaced the local modules with the nf-core modules you developed. I see that no file was actually removed from the outputs, but they are now being named using a different strategy. The md5sums from the files are all still matching, only that the method name is now not included in the output file's name.

Reply:
I think rather than deviating from the modules convention to use `meta.id` as default prefix, I think the name should be set in `modules.config` using `ext.prefix`.

Reply:
Agreed @grst